js/quotechew.js

```diff
@@ -113,8 +130,42 @@ var RE_SECTION_DELIM = /^[_-]{6,}$/;
 
 var RE_LIST_BOILER = /mailing list$/;
 
+/**
+ * Non-localized hard-coded heuristic to help us detect whether a line is the
+ * lead-in to a quote block.  This is only checked against the last content line
+ * preceding a quote block.  This only results in styling guidance, so if it's
+ * too aggresive or insufficiently aggressive, the worst that happens is the
+ * line will be dark gray instead of black.  (For now!  Moohoohahahahahaha.)
+ */
 var RE_WROTE_LINE = /wrote/;
 
+/**
```

Comment:
Yeah, this is strictly just maintaining the status quo right now.  Which was absolutely an English-specific hack.

Note that for localization we potentially don't actually want the locale to be providing these strings, but instead have compiled a database of all of the known different language phrasings for wrote among all the clients we can find.  Sortof like how browserbox has a database of folder names.

For example, when reading dev-fxos, you may note that French contributors use Thunderbird in the French locale and use "écrit" instead of wrote.  As an English user, I want the analysis logic to pick-up on that as being a quoting lead-in without having to change my locale to French, etc.

The existing locale strings we use for generating messages in our locale would indeed benefit from a cleanup that you propose, yeah.  Although the part where those strings are needed on the worker is still going to potentially get ugly or require the front-end to provide some hand-holding.  At least, I don't think we want l10n.js living in the worker like it owns the place and listening to all postMessaged messages to keep up-to-date with locale changes, etc.  But maybe with MessagePorts we can isolate the logic but still not have to hold its hand so much.